### ISSUE TITLE
Unused variable `first_digit_index`

### ISSUE THREAD

**Issue by chillenzer** [edbennett/format_multiple_errors#10]
[Here](https://github.com/edbennett/format_multiple_errors/blob/eed6f1044fd9d3150fc87d14066750944ca8dd5d/format_multiple_errors/formatter.py#L268), the value of `first_digit_index` is immediately overridden. You might want to use `_, decimal_places = ...` or `decimal_places = get_rounding_indices(...)[1]` instead.

**Comment by edbennett** [comment]
Good point—this change was introduced to placate pylint; before, the variable was not overridden but there was some duplicated logic that created too many variable names for the one function. When refactoring, I didn't spot that this redundancy was introduced.
